### PR TITLE
Initial commit for adding address line 2 to address verification

### PR DESCRIPTION
**Pull request by Ruskicowboy**
Integrated Address Line 2 and 3 into Address Verification

### CODE REVIEW

File: src/SimpleUPS/AddressValidate/Response.php [hunk 1]
```diff
@@ -115,12 +115,18 @@ public function fromXml(\SimpleXMLElement $xml)
...
             $address
                 ->setClassification($suggestedAddress->AddressClassification->Code)
-                ->setStreet($suggestedAddress->AddressLine)
+                ->setStreet($suggestedAddress->AddressLine[0])
                 ->setCity($suggestedAddress->PoliticalDivision2)
                 ->setStateProvinceCode($suggestedAddress->PoliticalDivision1)
                 ->setPostalCode($suggestedAddress->PostcodePrimaryLow)
                 ->setPostalCodeExtended($suggestedAddress->PostcodeExtendedLow)
                 ->setCountryCode($suggestedAddress->CountryCode);
+            if ($suggestedAddress->AddressLine[1]) {
```
Comment (bkuhl) [inline src/SimpleUPS/AddressValidate/Response.php]: Doesn't this need to be `if(isset())` ?

**Comment by bkuhl** [comment]
Thanks for submitting the PR!

File: src/SimpleUPS/AddressValidate/Response.php (hunk 1 above)
Comment (Ruskicowboy) [inline src/SimpleUPS/AddressValidate/Response.php]: Thanks.  I actually added if(!empty())

Comment (bkuhl) [inline src/SimpleUPS/AddressValidate/Response.php]: 👍 Thanks!

**Comment by bkuhl** [comment]
If you resolve conflicts we can get this merged in.

**Comment by Ruskicowboy** [comment]
Fixed conflicts

**Pull request merged**